python/ray/air/execution/tracked_actor.py

```diff
@@ -0,0 +1,36 @@
+from typing import Callable
+
+from ray.util.annotations import DeveloperAPI
+
+
+@DeveloperAPI
+class TrackedActor:
+    """Actor tracked by an actor manager.
+
+    This object is used to reference a Ray actor on an actor manager
+
+    Existence of this object does not mean that the Ray actor has already been started.
+    Actor state can be inquired from the actor manager tracking the Ray actor.
```

Comment:
Can you document the expected way for a user to create one of these, via `RayEventManager.add_actor()`?

Reply:
I can add an example

Reply:
Added